lib/bolt/cli.rb

```diff
@@ -516,7 +523,13 @@ def execute(options)
           code = Bolt::ProjectManager.new(config, outputter, pal).migrate
         end
       when 'lookup'
-        code = lookup(options[:object], options[:targets])
+        # Validate functions verifies one of these was passed
+        if options[:targets]
+          code = lookup(options[:object], options[:targets])
```

Comment:
This needs to be updated to accept vars like the plan lookup does.

Reply:
Good call I totally forgot to add this for non-plan-hierarchy lookups.